src/rpdk/core/contract/suite/contract_asserts.py

```diff
@@ -77,17 +77,6 @@ def response_does_not_contain_write_only_properties(resource_client, response):
     resource_client.assert_write_only_property_does_not_exist(response["resourceModel"])
 
 
-@decorate()
```

Comment:
how come it's redundant?

Reply:
because, we have added input == output for read handler response with create handler input [here](https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/suite/handler_create.py#L88-L90). Not sure we need to check read handler output with create handler output

Reply:
yes but this check asserts that during the execution the model does not change

Reply:
create  response model should match the read handler model. the model in the final progress of the create handler should match what is returned by read